firefox-ios/Client/TabManagement/TabManager.swift

```diff
@@ -65,6 +65,7 @@ protocol TabManager: AnyObject {
 
     /// Async Remove tab option using tabUUID.
     /// - Parameter tabUUID: UUID from the tab
+    @MainActor
```

Comment:
That was needed to fix the warning in the screenshot. The `TabManagerImplementation` `func removeTab(_ tabUUID: TabUUID) async` is already main actor, so this is just making the protocol main actor

<img width="2068" height="675" alt="Screenshot 2025-07-24 at 12 21 46 PM" src="https://github.com/user-attachments/assets/f6353ec0-2f69-4711-81c8-1821d446e3dd" />